tasks/integration_local_spec.rb

```diff
@@ -0,0 +1,74 @@
+desc 'Run integration test on `itamae local` command'
+task 'spec:integration:local' do
+  IntegrationLocalSpecRunner.new(
+    [
+      [
+        "spec/integration/recipes/default.rb",
+        "spec/integration/recipes/default2.rb",
+        "spec/integration/recipes/redefine.rb",
+        "spec/integration/recipes/local.rb",
+      ],
+      [
+        "--dry-run",
+        "spec/integration/recipes/dry_run.rb",
+      ],
+    ],
+    ['spec/integration/default_spec.rb']
+  ).run
+end
+
+class IntegrationLocalSpecRunner
+  CONTAINER_NAME = 'itamae'
+  include FileUtils
+
+  def initialize(suites, specs, ruby_version: RUBY_VERSION.split('.')[0..1].join('.'))
+    @suites = suites
+    @specs = specs
+    @ruby_version = ruby_version
+  end
+
+  def run
+    docker_run
+    prepare
+    provision
+    serverspec
+    clean_docker_container
+  end
+
+  def docker_run
+    mount_dir = Pathname(__dir__).join('../').to_s
+    sh 'docker', 'run', '--privileged', '-d', '--name', CONTAINER_NAME, '-v', "#{mount_dir}:/itamae", "ruby:#{@ruby_version}", 'sleep', '1d'
```

Comment:
`sleep` command is a hack...
`/sbin/init` is used for the existing integration test, but the image does not have `/sbin/init`.
So use `sleep` command instead.